src/app/clusters/bindings/bindings.cpp

```diff
@@ -121,38 +99,83 @@ bool emberAfBindingClusterBindCallback(app::CommandHandler * commandObj, const a
         return true;
     }
 
+    if (BindingManager::GetInstance().CreateBinding(fabricIndex, nodeId, localEndpoint, clusterId) != CHIP_NO_ERROR)
```

Comment:
Should we capture and log the error?

Otherwise the underlying error is lost (and on large systems that includes a file and line location) and we just get "ZCL_FAILURE" which would make debugging a real pain.